### ISSUE TITLE
Error in nnUNet_predict_image

### ISSUE THREAD

**Issue by kateduquesne** [wasserth/TotalSegmentator#198]
Dear

While running the example of the 'python_api' a problem occurs. I used the example code (while adding the specific task wanted):
```
from totalsegmentator.python_api import totalsegmentator
totalsegmentator(input_path, output_path,task="appendicular_bones")
```
The error occurs in the 'nnUNet_predict_image' more specifically in predictor.predict_from_files(dir_in, dir_out, save_probabilities=save_probabilities, overwrite=not continue_prediction,
                                 num_processes_preprocessing=npp, num_processes_segmentation_export=nps,
                                 folder_with_segs_from_prev_stage=prev_stage_predictions, 
                                 num_parts=num_parts, part_id=part_id)

I obtained the following error:
> Exception has occurred: RuntimeError       (note: full exception trace is shown but execution is paused at: <module>)
> 
>         An attempt has been made to start a new process before the
>         current process has finished its bootstrapping phase.
> 
>         This probably means that you are not using fork to start your
>         child processes and you have forgotten to use the proper idiom
>         in the main module:
> 
>             if __name__ == '__main__':
>                 freeze_support()
>                 ...
> 
>         The "freeze_support()" line can be omitted if the program
>         is not going to be frozen to produce an executable.
>   File ".../totalsegmentator/nnunet.py", line 210, in nnUNetv2_predict
>     predictor.predict_from_files(dir_in, dir_out,
>   File ".../totalsegmentator/nnunet.py", line 392, in nnUNet_predict_image
>     nnUNetv2_predict(tmp_dir, tmp_dir, task_id, model, folds, trainer, tta,
>   File ".../totalsegmentator/python_api.py", line 293, in totalsegmentator
>     seg_img = nnUNet_predict_image(input, output, task_id, model=model, folds=folds,
>   File ".../SegmentNewMexico.py", line 19, in <module>
>     totalsegmentator(input="...",
>   File "<string>", line 1, in <module> (Current frame)
> RuntimeError: 
>         An attempt has been made to start a new process before the
>         current process has finished its bootstrapping phase.
> 
>         This probably means that you are not using fork to start your
>         child processes and you have forgotten to use the proper idiom
>         in the main module:
> 
>             if __name__ == '__main__':
>                 freeze_support()
>                 ...
> 
>         The "freeze_support()" line can be omitted if the program
>         is not going to be frozen to produce an executable.

How can this be solved? Is the a totalsegmentator problem or a nnunet related problem?

**Comment by wasserth** [comment]
This seems like a nnunet related problem. Does this also happen if you call TotalSegmentator from the command line?

**Comment by kateduquesne** [comment]
Solved this problem similar to issue #190
def run():
    from totalsegmentator.python_api import totalsegmentator

    totalsegmentator(input="...", output="....",task="appendicular_bones")

if __name__ == '__main__':
    run()
